README.md

```diff
@@ -1,11 +1,12 @@
-# ComputeCpp SDK Readme
 
-![ComputeCpp™]
+<div align="center">
+  <br /><img src="https://developer.codeplay.com/assets/img/home/tools/computecppsdk.png"><br />
+</div>
 
+# ComputeCpp SDK Readme
+[![Build and Test](https://github.com/codeplaysoftware/computecpp-sdk/actions/workflows/build-test.yml/badge.svg)](https://github.com/codeplaysoftware/computecpp-sdk/actions/workflows/build-test.yml)
```

Comment:
Similarly, I'd hope that we can update the travis SVG link at the end of the file to the new shiny actions one you have here, which keeps things a bit more organised (while still leaving a useful link inline).

Reply:
I've also reverted this to the old style and updated the links!